arcade/application.py

```diff
@@ -915,6 +915,30 @@ def on_mouse_leave(self, x: int, y: int):
         """
         pass
 
+    def absolute(self, fractional_pos: tuple[float, float]) -> tuple[float, float]:
+        """Convert a relative fractional pair to absolute screen pixels.
+
+        As in the rest of arcade and OpenGL, the default coordinate system
+        places the origin at the bottom left.
+
+        :param fractional_pos: A position where the x and y are fractional
+            values of the window size.
+        :returns: A tuple of absolute pixel X and Y relative to bottom left.
+        """
```

Comment:
This docstring just seems to academic and lacks contex/example. It's unclear what "fractional pair" is. Also unsure is the method name itself is clear.